src/main/java/org/sagebionetworks/web/client/Portal.java

```diff
@@ -92,16 +93,12 @@ public void onSuccess() {
                 ginjector
                   .getSynapseProperties()
                   .getInitSynapsePropertiesFuture();
-              FluentFuture<Void> checkForUserChangeFuture = ginjector
-                .getAuthenticationController()
-                .getCheckForUserChangeFuture();
 
               FluentFuture
                 .from(
                   whenAllComplete(
                     featureFlagConfigFuture,
-                    synapsePropertiesFuture,
-                    checkForUserChangeFuture
```

Comment:
Big diff for a small change:

Do not execute `checkForUserChangeFuture` here. Run it later...